sw/host/tests/rom/e2e_bootup_bad_rom_ext_signature/src/main.rs

```diff
@@ -37,10 +38,15 @@ fn corrupt_rom_ext(rom_ext_path: &PathBuf) -> Result<NamedTempFile> {
         Err(error) => panic!("Problem reading the file: {:?}", error),
```

Comment:
Not part of your changes, but this could be have just been an `Err` instead of a panic, e.g.
```rust
let mut buffer = fs::read(rom_ext_path).context("failed to read ROM EXT file")?;
```
using `anyhow::Context`. Makes little difference, but allows it to be caught and might have a more readable message.

Reply:
fyi: #15531 removes this file